src/pharmpy/tools/iovsearch/tool.py

```diff
@@ -32,7 +32,7 @@
 
 def create_workflow(
     column: str = 'OCC',
-    list_of_parameters: Optional[List[str]] = None,
+    list_of_parameters: Optional[List[Union[str, List]]] = None,
```

Comment:
I think you should specify the type of the nested list, i.e. `Optional[List[Union[str, List[str]]]]`